src/test/coins_tests.cpp

```diff
@@ -24,8 +24,12 @@ class CCoinsViewTest : public CCoinsView
     std::map<uint256, bool> mapSerials_;
 
 public:
+    CCoinsViewTest() {
+        hashBestAnchor_ = ZCIncrementalMerkleTree::empty_root();
```

Comment:
I changed these things because I noticed the fixpoint was not tested properly in these tests.